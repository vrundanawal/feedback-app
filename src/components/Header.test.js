import { render } from "@testing-library/react";
import React from "react";
import { shallow } from "enzyme";
import Header from "./Header";

it("renders", () => {
  const { asFragment } = render(<Header text="Feedback UI" />);
  expect(asFragment()).toMatchSnapshot();
});

test("should test Header component", () => {
  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
});
