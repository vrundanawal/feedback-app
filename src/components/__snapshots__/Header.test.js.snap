// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <header
    style="background-color: rgba(0, 0, 0, 0.4); color: rgb(255, 106, 149);"
  >
    <div
      class="container"
    >
      <h2>
        Feedback UI
      </h2>
    </div>
  </header>
</DocumentFragment>
`;

exports[`should test Header component 1`] = `
<header
  style={
    Object {
      "backgroundColor": "rgba(0,0,0,0.4)",
      "color": "#ff6a95",
    }
  }
>
  <div
    className="container"
  >
    <h2>
      Feedback UI
    </h2>
  </div>
</header>
`;
